src/components/NameButton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+function NameButton() {
+  const storedName = localStorage.getItem("name");
+  console.log("storedName");
+  return (
+    <>
+      <h1>Welcome, {storedName}! </h1>
```

Comment:
Looks good ! usually names should go through a text-transform:capitalize just in case so names are shown cleanly

Reply:
Thanks for the sugession Onkar.